components/helpers/interfaces.tsx

```diff
@@ -2,10 +2,11 @@ import React from 'react'
 
 export type IFilter = 'All' | 'Open' | 'Voting' | 'Closed'
 export interface ContestType {
+  id: number
   name: string
   description: string
-  date_start: string
-  date_end: string
+  dateStart: string
+  dateEnd: string
   status: IFilter
 }
 export interface HomeProps {
```

Comment:
can we move the components props into the components files?

Reply:
this file should only be used for types or interfaces that are used in multiple places or that don't make sense to be in a specific file